AUTHENTICATION.md

```diff
@@ -85,7 +82,7 @@ $video = new VideoIntelligenceServiceClient([
 ]);
 ```
 
-However, some clients use the `keyFile` or `keyFilePath` option: 
+However, some clients use the `keyFile` or `keyFilePath` option:
```

Comment:
Only handwritten libraries have this client option. This should be removed or made more clear, or put at the very end. And it should also list the clients who support it, instead of just saying "some clients"